test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/HelloWorldEthereumSuite.java

```diff
@@ -493,7 +508,21 @@ private HapiApiSpec bigContractCreate() {
                                                                                                                         .aliasIdFor(
                                                                                                                                 SECP_256K1_SOURCE_KEY)
                                                                                                                         .getAlias()
-                                                                                                                        .toStringUtf8())))
+                                                                                                                        .toStringUtf8()))
+                                                                                        .evmAddress(
```

Comment:
Not critical for this PR, but I think we need to hide the details of constructing this expected `evmAddress` inside the `ContractFnResultAsserts`. 😁